OCM/Procotol/CustomViewProtocol.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CustomViewProtocol.swift
+//  OCM
+//
+//  Created by José Estela on 14/2/18.
+//  Copyright © 2018 Gigigo SL. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+//swiftlint:disable class_delegate_protocol
+
+/// This protocol allow the delegate to handle custom views needed by OCM
+/// - Since: 3.0.0
+public protocol CustomViewsDelegate {
```

Comment:
Wahooo! 👏Very nice! Although, I'd use CustomViewDelegate as a name, as the one you used on the file name (I know, I know, I won't ever stop with the naming complains)